cql3/statements/select_statement.cc

```diff
@@ -559,7 +560,9 @@ indexed_table_select_statement::do_execute_base_query(
     auto cmd = prepare_command_for_base_query(qp, options, state, now, bool(paging_state));
     auto timeout = db::timeout_clock::now() + get_timeout(state.get_client_state(), options);
     uint32_t queried_ranges_count = partition_ranges.size();
-    query_ranges_to_vnodes_generator ranges_to_vnodes(qp.proxy().get_token_metadata_ptr(), _schema, std::move(partition_ranges));
+    auto&& table = qp.proxy().local_db().find_column_family(_schema);
```

Comment:
nit: `qp.proxy().local_db()` can be shortened to `qp.db()`

Reply:
`qp.db()` returns data_ditctionary::database, which holds just the schema.